build.gradle.kts

```diff
@@ -676,7 +674,9 @@ fun printDockerCheckInfo() {
   val macDockerConnector = project.extra["macDockerConnector"] as? Boolean ?: false
   val isOrbStack = project.extra["isOrbStack"] as? Boolean ?: false
 
-  if (OperatingSystem.current().isMacOsX() &&
+  if (extra["skipDockerDependentTests"].toString().toBoolean()) {
+    project.extra["docker_it_test"] = false
```

Comment:
can you rename the "docker_it_test" to "docker_test"?

Reply:
Also for the tag name "gravitino-docker-it", can you please change to "gravitino-docker-test"?

Reply:
updated